rest-api-project/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const cors = require("cors");
+const zines = require("./zines.js");
+
+const app = express();
+
+app.use(cors());
+
+const PORT = 5001; 
+
+app.get("/", (req,res) => {
+    res.json("Hello, my name is 5001 and I will be your server this evening.");
+})
+
+app.get("/api/zines", (req,res) => {
+    res.json(zines);
+})
+
+app.post("/api/zines", (req,res) => {
+    let newZine = {
+        "title": req.body.title,
+        "author": req.body.author,
+        "category": req.body.category,
+        "format": req.body.format,
+        "pages": req.body.pages
+    };
+    zines.push(newZine);
+    return res.send("New zine has been seen, and saved!");
+    //res.json(zines);
```

Comment:
In general, the proper response to a POST request that is successful is to send back a 201 response with the object that was created, ideally with the new database ID. Since this app is not connected to a database, that is not possible, but you can still return `newZine` as a JSON object. All success messages should be handled by the front-end, as they may be different depending on the user's language, the device used to make the request, etc.